### DISCUSSION TITLE
Unable to open on garuda linux

### DISCUSSION THREAD

**Discussion by ooexiaoo** [Alex313031/thorium#253] [Q&A] [answered]
Hi, I tried installing it on Garuda Linux, its arch based, but I couldn't get the app image to run even after integrating it.
So I tried to install older versions but I just couldn't get it to launch.
I also tried to install it from arch repo and the same thing happened, after installing it won't launch.

Is there there something I need to do differently to get it to run?
I never had something like this happen to me on any other browsers, like Firefox, fire dragon, librewolf, brave.
Am I doing something wrong without knowing about it?

**Comment by gz83** [comment]
What processor are you currently using?

@ooexiaoo

**Reply by ooexiaoo** [reply]
Its an old I3 350m

**Reply by gz83** [reply, marked as answer]
Please use this version

https://github.com/Alex313031/Thorium-SSE3/releases/tag/M117.0.5938.157

@ooexiaoo

**Reply by ooexiaoo** [reply]
That worked, Thanks for getting back to me so quickly
Thank you so much 👍🏻
